docs-md/developer-guide/create-a-table.md

```diff
@@ -225,9 +257,9 @@ to include a function like COUNT(*) in the SELECT clause.
 
 ```sql
 CREATE TABLE pageviews_table AS
-  SELECT viewtime, userid, pageid, COUNT(*) AS TOTAL
+  SELECT userid, pageid, COUNT(*) AS TOTAL
   FROM pageviews_original WINDOW TUMBLING (SIZE 1 MINUTES)
-  GROUP BY viewtime, userid, pageid
+  GROUP BY userid, pageid
   EMIT CHANGES;
```

Comment:
Removed `viewtime` from key of aggregation.

IMHO, having `viewtime` in there makes it a nonsensical example.  

Without `viewtime` the table being built counts the number of times a specific user visits each page within a single minute.   OK, that's a pretty wacky table, but it at least works.

With `viewtime` the table being built counts the number of times a specific user visits each page _within the same millisecond_ and then windowed it per minute.  It's highly unlikely a user can visit the same page twice within the same millisecond, so the count will always be 1. 

Hence, removed `viewtime`.